tracer/build/_build/Build.GitHub.cs

```diff
@@ -84,6 +86,88 @@ partial class Build
             Console.WriteLine($"PR assigned");
         });
 
+    Target AssignLabelsToPullRequest => _ => _
+       .Unlisted()
+       .Requires(() => GitHubRepositoryName)
+       .Requires(() => GitHubToken)
+       .Requires(() => PullRequestNumber)
+       .Executes(async() =>
+        {
+            var client = GetGitHubClient();
+
+            var pr = await client.PullRequest.Get(
+                owner: GitHubRepositoryOwner,
+                name: GitHubRepositoryName,
+                number: PullRequestNumber.Value);
+
+            GitTasks.Git("fetch origin master:master", logOutput: false);
```

Comment:
I think we should remove this line? Seems unnecessary given you've already done a checkout?

```suggestion
            GitTasks.Git("fetch origin master:master", logOutput: false);
```

Reply:
So my first try was without this line. And it failed `fatal: ambiguous argument 'master': unknown revision or path not in the working tree.`. 
This is me fixing the issue after a quick look at StackOverflow :p So I'd happily change to a better solution (or at least adding a comment)

Reply:
ah, the next line should probably `origin/master`. The fetch here is creating a local `master` branch and associating it with the upstream, which you don't want/need to do 🙂

Reply:
Thanks a lot. Fixed in 11c3cb345eb279dba2a0df0166e1e845a683e54d

Reply:
Actually, it didn't work. I've roll backed and will merge :) I can revisit later